tests/unit/lms/services/grading_info_test.py

```diff
@@ -50,31 +54,29 @@ def test_it_returns_an_empty_list_if_there_are_no_matching_GradingInfos(
         assert list(grading_infos) == []
 
     @pytest.fixture(autouse=True)
-    def matching_grading_infos(self, db_session):
+    def matching_grading_infos(self):
         """Add some GradingInfo's that should match the DB query in the test above."""
-        matching_grading_infos = [self.grading_info("matching", i) for i in range(3)]
-        db_session.add_all(matching_grading_infos)
-        return matching_grading_infos
+        matching_grading_info = partial(
+            factories.GradingInfo,
+            oauth_consumer_key="matching_oauth_consumer_key",
+            context_id="matching_context_id",
+            resource_link_id="matching_resource_link_id",
+        )
+
+        return [
+            matching_grading_info(),
+            matching_grading_info(),
+            matching_grading_info(),
+        ]
```

Comment:
I've fixed this locally to but GitHub won't let me push it. Should be:

```python
        return factories.GradingInfo.create_batch(
            size=3,
            oauth_consumer_key="matching_oauth_consumer_key",
            context_id="matching_context_id",
            resource_link_id="matching_resource_link_id",
        )
```

Reply:
Yeah that's a lot nicer